### PR TITLE
State: Add tests for all of the UI actions

### PR DESCRIPTION
**Pull request by tyxla**
This PR adds tests for all of the UI actions, and expands the current tests to cover more use cases. It also does some minor code style improvements, like sorting imports and tests alphabetically.

To test:
* Checkout this branch
* Verify all UI actions tests pass by running:

```
npm run test-client client/state/ui/test/actions.js 
```

### CODE REVIEW

File: client/state/ui/test/actions.js [hunk 1]
```diff
@@ -7,38 +7,125 @@ import { expect } from 'chai';
...
 			expect( action ).to.eql( {
 				type: SELECTED_SITE_SET,
-				siteId: 2916284
+				siteId: null
+			} );
+		} );
+	} );
+
+	describe( 'setPreviewShowing()', () => {
+		it( 'should return an action object where isShowing equals to true', () => {
```
Comment (sirreal) [inline client/state/ui/test/actions.js]: not `equals to`, just `equals`, or better yet `isShowing is true`.

File: client/state/ui/test/actions.js [hunk 2]
```diff
@@ -7,38 +7,125 @@ import { expect } from 'chai';
...
+			const action = setPreviewShowing( true );
+
+			expect( action ).to.eql( {
+				type: PREVIEW_IS_SHOWING,
+				isShowing: true
+			} );
+		} );
+
+		it( 'should return an action object where isShowing equals to false', () => {
```
Comment (sirreal) [inline client/state/ui/test/actions.js]: not `equals to`, just `equals`, or better yet `isShowing is false`.

File: client/state/ui/test/actions.js [hunk 3]
```diff
@@ -7,38 +7,125 @@ import { expect } from 'chai';
...
 				type: SELECTED_SITE_SET,
-				siteId: 2916284
+				siteId: null
+			} );
+		} );
+	} );
+
+	describe( 'setPreviewShowing()', () => {
+		it( 'should return an action object where isShowing equals to true', () => {
+			const action = setPreviewShowing( true );
```
Comment (sirreal) [inline client/state/ui/test/actions.js]: Aside: I wish there were `previewShow` and `previewHide` actions.

File: client/state/ui/test/actions.js [hunk 4]
```diff
@@ -7,38 +7,125 @@ import { expect } from 'chai';
...
 			} );
 		} );
 	} );
 
 	describe( 'setRoute()', () => {
-		it( 'should return an action object', () => {
-			const action = setRoute( '/foo' );
+		const route = '/foo';
+
+		it( 'should return an action object with path and no query arguments', () => {
```
Comment (sirreal) [inline client/state/ui/test/actions.js]: Maybe `should return and action with an empty query object if no query is supplied`?

File: client/state/ui/test/actions.js (hunk 3 above)
Comment (tyxla) [reply client/state/ui/test/actions.js]: That's too nitpicky even for you :laughing:

File: client/state/ui/test/actions.js (hunk 1 above)
Comment (tyxla) [reply client/state/ui/test/actions.js]: :+1: c8ab6f317f174ec44e50af301a0f36f53a2d4cad

File: client/state/ui/test/actions.js (hunk 2 above)
Comment (tyxla) [reply client/state/ui/test/actions.js]: :+1: c8ab6f317f174ec44e50af301a0f36f53a2d4cad

File: client/state/ui/test/actions.js (hunk 4 above)
Comment (tyxla) [reply client/state/ui/test/actions.js]: :+1: c8ab6f317f174ec44e50af301a0f36f53a2d4cad

**Comment by tyxla** [comment]
Thank you Jon :+1:

**Pull request merged**